packages/menu/src/Menu.tsx

```diff
@@ -97,12 +97,21 @@ function Menu({
 
   const hasSetInitialFocus = useRef(false);
   const hasSetInitialOpen = useRef(false);
+  const [focused, setFocused] = useState<HTMLElement>(refs[0] || null);
```

Comment:
Drive by, just moving some code around that increased my readability. functionality is the same

Reply:
Should we run a changeset against this, or just wait until we update Menu further to publish?

Reply:
This has no functional change at all, so can just wiat.